doc/man3/OCSP_resp_find_status.pod

```diff
@@ -105,10 +100,11 @@ signed the response are known via some out-of-band mechanism.
 OCSP_resp_get0_id() gets the responder id of I<bs>. If the responder ID is
 a name then <*pname> is set to the name and I<*pid> is set to NULL. If the
 responder ID is by key ID then I<*pid> is set to the key ID and I<*pname>
-is set to NULL. OCSP_resp_get1_id() leaves ownership of I<*pid> and I<*pname>
-with the caller, who is responsible for freeing them. Both functions return 1
-in case of success and 0 in case of failure. If OCSP_resp_get1_id() returns 0,
-no freeing of the results is necessary.
+is set to NULL.
+OCSP_resp_get1_id() leaves ownership of I<*pid> and I<*pname> with the caller,
+who is responsible for freeing them.
+Both functions return 1 in case of success and 0 in case of failure.
```

Comment:
I'd include this text in the paragraph for both functions.  I did this above for the get1.  For get0:

> _This function returns 1 on success and 0 on failure._

However, this could be put in the return values section where we'd normally have it.  Likewise for the get1, although I'm not sure if the "don't need to free on failure" belongs there.

Reply:
Indeed preferable - in this case also simpler - to move the description of the return value to the `RETURN VALUES` section. Done.
The "don't need to free on failure" pertains to the `get1` variant and can still be kept there.